amplify/backend/function/putUser/src/index.py

```diff
@@ -16,22 +16,28 @@ def lambda_handler(event, context):
     # Get vertex ID
     query = f"g.V().has('username', '{username}').id();"
     vertexID = db.submit(query).all().result()
```

Comment:
Check for vertexID here and exit early, so we don't have to loop over the attributes if the user doesn't exists.

```suggestion
    vertexID = db.submit(query).all().result()
    if len(vertexID) === 0:
      result =  f"Server failed to find user : {username}"
      statusCode = 404
      response = {
        "isBase64Encoded": False,
        "statusCode": statusCode,
        "headers": {},
        "multiValueHeaders": {},
        "body": json.dumps(result)
      }
      return response
```